data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1228 @@
+		system "Kneirc"
+		fleet
+			names "gegno scin"
+			variant
+				"Coesite" 3
+	npc
+		on provoke
+			set "zydee interference"
+		government "Gegno Scin"
+		personality heroic entering
+		system "Kneirc"
+		fleet
+			names "gegno scin"
+			variant
+				"Feldspar"
+				"Felsic" 3
+				"Coesite" 4
+				"Halite" 6
+
+
+mission "Curious Civilians"
+	invisible
+	landing
+	to offer
+		has "First Contact: Gegno Scin: offered"
+		not "Gegno Genocide Defense: offered"
+	to complete
+		never
+	on enter
+		system
+			government "Gegno"
+		fail
+		"gegno: encounters" ++
+		event "gegno: civilian suspicion"
+
+
+mission "Gegno Suspicions"
+	invisible
+	landing
+	source "Giaru Gegno"
+	to offer
+		"gegno: encounters" >= 3
+		"gegno: encounters" < 6
+		not "Gegno Genocide Defense: offered"
+	on offer
+		conversation
+			`After you go through your post-docking routine and enter the ringworld, you can't help but feel that something is a bit off. Normally, the Quarg here would be swiftly walking about, focused on their duties, but for some reason, there seem to be fewer of them around than usual. Moreover, those you do come across are occasionally having discussions with one another, sometimes looking in your direction even for the slightest moment. What stands out the most is the utter lack of any Gegno in the area you're visiting.`
+			`	A few minutes later, while you are making your way through the pristine interior of the ring, you hear a voice call to you from down one of the various branching corridors. Turning around, you recognize the unique attire of the Quarg who has helped you in the past, along with a few others. "Circumstantial greetings once more, <first> <last>. Might we have a significant moment of your attention?"`
+			choice
+				`	"Sure."`
+					goto "follow"
+				`	"How long will it take?"`
+			`	"As long as is required, for this matter is of symbolic importance," it says, with a slightly elevated tone.`
+
+			label follow
+			`	You follow the group of Quarg to a smaller discussion room, similar to the one you've previously visited. Or perhaps you've been visiting a different room each time, considering how unfathomably large a ringworld is. Once you sit down, the Quarg from before looks at you for a moment, looks down at the table, then back up to you.`
+			
+			branch "negative rep"
+				or
+					"reputation: Gegno Vi" < 0
+					"reputation: Gegno Scin" < 0
+
+			`	"Formerly, I have offered suggestions of heedful interplay regarding the Gegno. We have been alerted of a few additional encounters you've experienced, although they seemed minimal in nature. In such cases, it has been generally described that you left in a timely manner, for which we are notably appreciative. However," the Quarg continues, "despite your efforts in recommended cautions, there are some Gegno figures of meaningful standing that expressed frustration in your trivial appearances. They inquired of us as to the possibility of unnecessary reoccurrences."`
```

Comment:
```suggestion
			`	"Formerly, I have offered suggestions of heedful interplay regarding the Gegno. We have been alerted of a few additional encounters you have experienced, although they seemed minimal in nature. In such cases, it has been generally described that you left in a timely manner, for which we are notably appreciative. However," the Quarg continues, "despite your efforts in recommended cautions, there are some Gegno figures of meaningful standing that expressed frustration in your trivial appearances. They inquired of us as to the possibility of unnecessary reoccurrences."`
```
If you're going for the "archaic" English approach with the Quarg like in the Korath ringworld, I think avoiding contractions makes sense here.